roles/foreman_postgresql/tasks/main.yml

```diff
@@ -21,10 +21,18 @@
     owner: postgres
     group: postgres
 
+- name: Set listen addresses to *
```

Comment:
Should we make this optional? For all-in-one, we may not want to listen everywhere.

Reply:
How would you feel about leaving this and filing an issue or todo to support running on localhost?